### ISSUE TITLE
Show symbol relative to file type and extension

### ISSUE THREAD

**Issue by FrankPujo** [softvar/enhanced-github#84]
Watching a repository is kinda boring because the icons are just either a file icon or a folder icon. My idea was to add an icon for each file type (at least for the most common languages). To give an overview just open GitLab and look at the huge difference.

**Comment by Cokaps** [comment]
I agree with you, there is a [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=PKief.material-icon-theme) which have similar idea (for VSCode) and there are some icons there, may be you could have a look at that, too
